extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,257 @@
+        },
+        "https://ns.adobe.com/experience/campaign/message/delivery/id": {
+          "title": "Activity Identifer",
+          "type": "string"
+        },
+        "https://ns.adobe.com/experience/campaign/message/delivery/title": {
+          "title": "Title",
+          "type": "string",
+          "description": "Human readable name of the activity.",
+          "maxLength": 128
+        },
+        "https://ns.adobe.com/experience/campaign/message/delivery/communicationchannel": {
+          "title": "Channel",
+          "$ref": "https://ns.adobe.com/xdm/channels/channel",
+          "description": "The marketing channel used to execute this activity."
+        },
+        "https://ns.adobe.com/experience/campaign/message/delivery/description": {
+          "title": "Description",
+          "type": "string",
+          "description": "Description of the activity intent."
+        },
+        "https://ns.adobe.com/experience/campaign/message/delivery/trackingcode": {
+          "title": "Tracking Code",
+          "type": "string",
+          "description": "A human readable tracking code for this activity. A same tracking code can be reused over multiple activities. For instance this could be a promotion code. That same code could be used over different channels at different point of time. All activities sharing that same promotion code can then be grouped together within a same report even if they are not under the same campaign."
+        },
+        "https://ns.adobe.com/experience/campaign/message/delivery/startdate": {
+          "title": "Start Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "The intended start date of the activity. In Adobe Campaign, the start date is when the delivery of the messages is scheduled to start."
+        },
+        "https://ns.adobe.com/experience/campaign/message/delivery/enddate": {
+          "title": "End Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "The intended end date of the activity. In Adobe Campaign, the end date is the expiration date of the delivery. After this date, no messages are expected to be delivered. Unlike the `startDate`, the `endDate` is not required. Some activities can be *open* and so the `endDate` is not known at the time of the creation of the entity (e.g., a real-time messaging activity in Adobe Campaign)."
+        },
+        "https://ns.adobe.com/experience/campaign/message/delivery/messagesclass": {
+          "title": "Message Class",
+          "type": "string",
+          "description": "Marketing classification of the messages sent through that delivery.",
+          "default": "bulk",
+          "enum": [
+            "bulk",
+            "automation",
+            "transactional"
+          ],
+          "meta:enum": {
+            "bulk": "Bulk",
+            "automation": "Automation",
```

Comment:
What's the difference between `automation` and `transactional`?

Reply:
`automation` refers to deliveries that are recurring in nature.
`transactional` are deliveries that are triggered through an external/internal event.

Reply:
then let's call them what they are, please, @saurabhere 

- recurring
- event-based